backend/src/main/java/com/bakdata/conquery/models/config/FormBackendConfig.java

```diff
@@ -100,13 +104,35 @@ public void initialize(ManagerNode managerNode) {
 		managerNode.getFormScanner().registerFrontendFormConfigProvider(this::registerFormConfigs);
 	}
 
+	/**
+	 * Retrieves the version information from the form backend and writes it to the {@link VersionInfo}
+	 */
+	private void updateVersion(ExternalFormBackendApi externalApi) {
+
+		try {
+			final String version = externalApi.getVersion();
```

Comment:
ok, der Endpunkt soll einfach die version ausgeben. Steffen hatte mich schon gefragt in deiner abwesenheit, und wir hatten  `{ "version" : ... }` gewählt. Ist aber vermutlich sogar cleaner was du vorschlägst

Reply:
Ich habe jetzt Steffens Vorschlag implementiert, nur unwrap ich das Objekt schon in der Api-Klasse auf unserer Seite